index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
- [ ] [OPTIONAL] This is a great milestone to ensure that your project has a `title`. However, this is a default title. To ensure that the project is more accessible to screen readers, consider customizing the title according to the project name.